src/Payum/Core/Bridge/Propel/Storage/Propel1Storage.php

```diff
@@ -0,0 +1,100 @@
+<?php
+namespace Payum\Core\Bridge\Propel\Storage;
+
+use Payum\Core\Storage\AbstractStorage;
+use Payum\Core\Exception\LogicException;
+use Payum\Core\Model\Identity;
+
+class Propel1Storage extends AbstractStorage
+{
+    /**
+     * @var string
+     */
+    protected $modelQuery;
+    /**
+     * @var string
+     */
+    protected $modelPeer;
+
+    /**
+     * @param string $modelClass
+     * @param string $modelPeer
+     * @param string $modelQuery
+     */
+    public function __construct($modelClass, $modelPeer = null, $modelQuery = null)
+    {
+        parent::__construct($modelClass);
+
+        $this->modelQuery = $modelQuery ?: $modelClass.'Query';
+        $this->modelPeer = $modelPeer ?: $modelClass.'Peer';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function findBy(array $criteria)
+    {
+        throw new LogicException('Method is not supported by the storage.');
```

Comment:
is it really not supported? It is hard to implement such method for filesystem storage but should be possible for propel.

(we can do it as separate PR)